CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+## Install Cultionet
+
+### Clone the latest repo
+
+```commandline
+git clone https://github.com/jgrss/cultionet.git
+```
+
+### Create a virtual environment
+
+```commandline
+pyenv virtualenv 3.8.15 venv.cnet
```

Comment:
thanks for adding in this file. You have `3.8.12` in ReadME I think. It seems like `3.8.15` has some few tweaks https://docs.python.org/release/3.8.15/whatsnew/changelog.html

Reply:
Thanks for catching this. We've been using `3.8.15` so I'll update the README.